File: pythag1224.c
/*
C version
Compile command: gcc pythag1224.c -o pythag1224 -lm
*/
#include <stdio.h>
#include <math.h>

void main(int argc, char* argv[])
{
    int area = 1224;
    for(int a = 1; ; a++) {
        int b = area*2/a;
        float c = sqrt(a*a + b*b);
        if (c == floor(c)) {
            printf("%d %d %d\n", a, b, (int)c);
            break;
        }
    }
}

File: pythag1224.js
let A = 1224;
let found = false;
for (let a=1; found == false; a=a+1) {
    let b = A*2/a
    let c = Math.sqrt(a*a + b*b);
    if (c === Math.floor(c)) {
        console.log(`${a} ${b} ${c}`);
        found = true;
    }
}